server/routes/overview/mappers/timelineMapper.test.ts

```diff
@@ -0,0 +1,83 @@
+import type { TimelineResponse, TimelineEventResponse } from 'educationAndWorkPlanApiClient'
+import type { Timeline, TimelineEvent } from 'viewModels'
+import { toTimeline, toTimelineEvent } from './timelineMapper'
+
+describe('timelineMapper', () => {
+  it('should map to Timeline', () => {
+    // Given
+    const timeline: TimelineResponse = {
+      problemRetrievingData: false,
```

Comment:
This is not _quite_ right. A `TimelineResponse` object is the API object, straight out of the REST API. It does not have a `problemRetrievingData` field. It's only our view models that have this field, not the REST API types.
Just nuke this line (line 9) and it'll be OK 👍 

(but I'm about to make the same comment on the test data builder as well)

Reply:
Good spot I've resolved this